packages/xo-server/src/xo-mixins/proxies.js

```diff
@@ -16,6 +16,10 @@ import patch from '../patch'
 import readChunk from '../_readStreamChunk'
 import { generateToken } from '../utils'
 
+const IP_REG = /^\d+\/ip$/
```

Comment:
Do you know the relationship between `1/ip` and `1/ipv4/0`? Is there always a `n/ip` field?

Reply:
If a `VM` has an IP it will have a [`n/ip` field](https://discussions.citrix.com/topic/351974-xenserver-guest-metrics-ip-address/). This properties are not documented, so, i don't have more information about it.

Reply:
Ok.